### PR TITLE
Update  jupyter outputs to  default to port-forwarding

### PR DESCRIPTION
**Pull request by umeshkumhar**
- Update jupyter outputs to port-forwarding page
- update MP suggestedNext Steps

### CODE REVIEW

File: applications/jupyter/main.tf [hunk 1]
```diff
@@ -58,6 +58,7 @@ locals {
 
 locals {
   workload_identity_service_account = var.goog_cm_deployment_name != "" ? "${var.goog_cm_deployment_name}-${var.workload_identity_service_account}" : var.workload_identity_service_account
+  jupyterhub_default_uri            = "https://console.cloud.google.com/kubernetes/service/${var.cluster_location}/${var.cluster_name}/${var.kubernetes_namespace}/proxy-public/overview?project=${var.project_id}"
```
Comment (imreddy13) [inline applications/jupyter/main.tf:61]: nit: can we merge the locals blocks on lines 50 and 59?

File: applications/jupyter/outputs.tf [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 output "jupyterhub_uri" {
   description = "JupyterHub Endpoint to access user interface. In case of private IP, consider port-forwarding."
-  value       = "http://${module.jupyterhub.jupyterhub_uri}"
+  value       = module.jupyterhub.jupyterhub_uri != "" ? "http://${module.jupyterhub.jupyterhub_uri}" : local.jupyterhub_default_uri
```
Comment (imreddy13) [inline applications/jupyter/outputs.tf:17]: Can you please update RAG and Jupyter READMEs with how to use this? We need to mention port forwarding before logging in with dummy credentials.

**Review by imreddy13 — APPROVED** [review]

File: applications/jupyter/main.tf (hunk 1 above)
Comment (umeshkumhar) [reply applications/jupyter/main.tf:61]: in next PR

File: applications/jupyter/outputs.tf (hunk 2 above)
Comment (umeshkumhar) [reply applications/jupyter/outputs.tf:17]: will be taken care of in https://github.com/GoogleCloudPlatform/ai-on-gke/pull/384

**Pull request merged**